### PR TITLE
3.18.0

### PR DESCRIPTION
**Pull request by greg0ire**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.18.0](https://github.com/sonata-project/SonataCoreBundle/compare/3.17.2...3.18.0) - 2020-01-28
+### Changed
+Switch to phpunit 8 setUp signature in tests
```
Comment (OskarStark) [inline CHANGELOG.md]:
```suggestion
- Switch to phpunit 8 setUp signature in tests
```

**Comment by core23** [comment]
Is it worth a release? It's just for the tests and this bundle is deprecated

**Comment by greg0ire** [comment]
@core23 it's just for us, to help use migrating to phpunit8… maybe not worth it, but I don't think we should care a lot about that since the bundle is deprecated.

**Comment by core23** [comment]
If you all are okay with it, feel free to tag this.

**Pull request merged**